src/nutate.jl

```diff
@@ -90,19 +90,20 @@ all terms larger than 0.0003".
 (1) Find the nutation in longitude and obliquity 1987 on Apr 10 at Oh.  This is
 example 22.a from Meeus
 
-``` julia
-jd = jdcnv(1987, 4, 10);
-nutate(jd)
-# => (-3.787931077110755,9.442520698644401)
+```jldoctest
+julia> jd = jdcnv(1987, 4, 10);
+
+julia> nutate(jd)
+(-3.7879310771107564, 9.4425206986444)
```

Comment:
All incorrect results except this have been updated.
Even after making this branch up to date with master, I'm still getting
```julia
julia> nutate(jdcnv(1987, 4, 10))
(-3.7879310771107564, 9.4425206986444)
```
as the result.
Could the difference in the version of our local Julia installation be the cause of the difference?